python/marvin/contrib/vacs/mangaVM.py

```diff
@@ -0,0 +1,77 @@
+# !usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Licensed under a 3-clause BSD license.
+#
+# @Author: Brian Cherinka
+# @Date:   2018-10-11 17:51:43
+# @Last modified by:   Brian Cherinka
+# @Last Modified time: 2018-11-29 17:23:15
+
+from __future__ import print_function, division, absolute_import
+
+import numpy as np
+import astropy
+import marvin.tools
+
+
+from .base import VACMixIn
+
+
+class VMORPHOVAC(VACMixIn):
+    """Provides access to the MaNGA-VISUAL-MORPHOLOGY VAC.
+
+    VAC name: manga_visual_morpho
+
+    URL: link
+
+    Description: Returns visual morphology data
+
```

Comment:
Can you write a brief description for your VAC here, and change the URL link to point to the DR15 documentation page for your VAC?

Reply:
I did write the url where I think my datamodel will be placed according to the information given by Joel. Even when my VAC content is DR15, it will be placed as part fo the new DR16.

Reply:
Do you mind to have a look at the new version, please?

Reply:
I will have a look.